docs/source/tutorials/Creating_gap_tracks.md

```diff
@@ -0,0 +1,28 @@
+# Creating gap tracks
+
+We use a little python script from [sequencetools repository](https://github.com/lexnederbragt/sequencetools). NB: If you use this script in an article you need to proprely aknowledge Lex Nederbragt.
```

Comment:
Add some explanation/background at the top so that people understand the why

Reply:
For instance, please mention what a track is.

Reply:
added explanation
removed track in here -> this is more IGV vocabulary